core/templates/dev/head/collection_player/collection_player.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base.html" %}
+
+{% block subtitle %}
+  {{ collection_title }}
+{% endblock subtitle %}
+
+{% block header_js %}
+  {{ super() }}
+  {% if collection_id %}
+    <script type="text/javascript">
+      GLOBALS.collectionId = JSON.parse('{{collection_id|js_string}}');
+      GLOBALS.isLoggedIn = Boolean(JSON.parse('{{is_logged_in|js_string}}'));
+    </script>
+
+    <style>
+      html, body {
+        background-color: #eee;
+      }
+    </style>
+  {% endif %}
+
+  {{dependencies_html}}
+{% endblock header_js %}
+
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span class="protractor-test-collection-header" itemprop="description">{{collection_title}}</span>
```

Comment:
There's some humor to this, I think. I'm pretty sure I or someone else removed the '> Explore' part from the breadcrumbs sometime in the summer for some reason (can't remember why at the moment). I'm pretty sure it was there before I started on Oppia; it might be good to see if any of us remember if it was there and, if so, why it was removed before adding it back.